app/views/sessions/new.html.erb

```diff
@@ -6,8 +6,8 @@
     <%= form_for(:session, url: login_path) do |f| %>
       <%= f.label :email %>
       <%= f.email_field :email, class: "form-control" %>
-      <%= f.label :password %>
-      <%= f.password_field :pass, class: "form-control" %>
+      <%= f.label :pass %>
```

Comment:
password?

Reply:
ý là ghi cho nó rõ tên biến ra ấy thanh niên ạ

Reply:
what do you mean? I don't  understand.
ý chị là em không nên đặt cái biến là pass à chị. theo em nghĩ là biến chỉ tượng trưng thôi mà ạ